test/sanitizer.test.js

```diff
@@ -0,0 +1,57 @@
+/* global describe, it('should */
+'use strict'
+
+const bindings = require('../')
+const expect = require('chai').expect
+
+const Sanitizer = bindings.Sanitizer
+
+describe('sanitizer', function () {
+  it('should sanitizes an insert list', function () {
+    const sql = "INSERT INTO `queries` (`asdf_id`, `asdf_prices`, `created_at`, `updated_at`, `blue_pill`, `yearly_tax`, `rate`, `steam_id`, `red_pill`, `dimitri`, `origin`) VALUES (19231, 3, 'cat', 'dog', 111.0, 126.0, 116.0, 79.0, 72.0, 73.0, ?, 1, 3, 229.284, ?, ?, 100, ?, 0, 3, 1, ?, NULL, NULL, ?, 4, ?)"
+    const result = Sanitizer.sanitize(sql)
+    expect(result).equal("INSERT INTO `queries` (`asdf_id`, `asdf_prices`, `created_at`, `updated_at`, `blue_pill`, `yearly_tax`, `rate`, `steam_id`, `red_pill`, `dimitri`, `origin`) VALUES (0, 0, '?', '?', 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, ?, 0, 0, 0.0, ?, ?, 0, ?, 0, 0, 0, ?, NULL, NULL, ?, 0, ?)")
+  })
+
+  it('should sanitizes a in list', function () {
+    const sql = "SELECT \"game_types\".* FROM \"game_types\" WHERE \"game_types\".\"game_id\" IN (1162)"
+    const result = Sanitizer.sanitize(sql)
+    expect(result).equal("SELECT \"?\".* FROM \"?\" WHERE \"?\".\"?\" IN (0)")
```

Comment:
I don't think the field names should be sanitized only the parameters

Reply:
True. All instrumented packages that use the Sanitizer, also use the `Sanitizer.OBOE_SQLSANITIZE_KEEPDOUBLE` option and thus keep field names (see  the two similar tests on lines 28 and 34). But, since the option to `OBOE_SQLSANITIZE_DROPDOUBLE` is the default, we also test that.

Generally, test coverage here is just meant to "lock" how things are (and have always been) done (correct or not). If someone, someday, decides to move away from this Sanitizer and implement something simpler (à la Ruby), the change will be clearer.